setup.py

```diff
@@ -49,7 +49,7 @@
             'flake8',
         ],
         coverage=[
-            'pytest-cov',
+            'coverage',
```

Comment:
Recent versions of ``pytest-cov`` provide a good integration of Coverage into Py.test 
  and I think it makes sense to keep it.